test/io/fixtures.yaml

```diff
@@ -153,6 +153,41 @@ roleclaims:
       role: postgrest_test_author
       other: true
     expected_status: 401
+  - key: '.realm_access.roles[?(@ == "postgrest_test_author")]'
```

Comment:
Note: These values get used on this test https://github.com/PostgREST/postgrest/blob/60d92f64dcab681ffc1674cb87935cb0a638603b/test/io/test_io.py#L191-L205

Reply:
Yes, exactly! That's why I have added them here to test the feature.